docs/matching.rst

```diff
@@ -22,6 +22,39 @@ For a simple match by label and property::
         >>> matcher.match("Person", name="Keanu Reeves").first()
         (_224:Person {born:1964,name:"Keanu Reeves"})
 
+Many types of comparisons can be made with operator suffixes. By appending the opeartor suffix you change the operator used in Cypher expression.
+
+        +--------------------------+--------------+-------------+------------------------------------------------------------+
+        | Description              | Suffix       | Operator    | Example                                                    |
+        +--------------------------+--------------+-------------+------------------------------------------------------------+
+        | Explicit Equal           | __exact      | =           | >>>matcher.match("Person", name__exact="Kevin Bacon")         |
+        |                          |              |             | MATCH (_:Person) WHERE name = "Kevin Bacon" RETURN _       |
+        +--------------------------+--------------+-------------+------------------------------------------------------------+
+        | Not Equal                | __not        | <>          | matcher.match("Person", name__not="Rick Astley")           |
```

Comment:
Bonus points for Rick Astley. Although some of his new stuff has been pretty good :)